include/daScript/simulate/aot_builtin_jobque.h

```diff
@@ -102,7 +102,7 @@ namespace das {
             }
             lock_guard<mutex> guard2(that->mCompleteMutex);
             for ( auto & f : pipe ) {
-                that->pipe.emplace_back(move(f));
+                that->pipe.emplace_back(std::move(f));
```

Comment:
just move(f) please. All std related functions should be visible from das namespace.
We use eastl instead of default std and all this std:: will break our build.
Same problem in some next changes.

Reply:
Oh, OK. I added that to silence a warning, should I just add the relevant `-W-no-xxx`?